code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityMapTests.java

```diff
@@ -337,7 +337,7 @@ public void testFromXDMMap_InvalidXDMData() throws Exception {
 		IdentityMap map = IdentityMap.fromXDMMap(xdmData);
 
 		// verify
-		assertTrue(map.isEmpty());
+		assertNull(map);
```

Comment:
@kevinlind @emdobrin Should we send null or empty map. Here we send null in case of errors https://github.com/adobe/aepsdk-edgeidentity-android/pull/51/files#diff-f845ec51b3fefa9868ac6722831d0c2c64f2fae51587b26485f0acbf4a76d479R294. I think we should either have null in case of error or empty Map. So right now the PR returns null in all error cases